File: src/web/src/app/model/vehicle.ts
export class Vehicle{
  vehicleId?: number;
  brand?: string;
  model?: string;
  vin?: string;
  year?: number;
  price?: number;

  constructor(
    vehicleId?: number,
    brand?: string,
    model?: string,
    vin?: string,
    year?: number,
    price?: number
  ) {
      this.vehicleId = vehicleId;
      this.brand = brand;
      this.model = model;
      this.vin = vin;
      this.year = year;
      this.price = price;
    }
}

File: src/main/java/allane/model/Vehicle.java
package allane.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Vehicle {
    @Id
    @GeneratedValue(strategy= GenerationType.AUTO)
    @Column(name = "vehicle_id")
    private long vehicleId;
    private String brand;
    private String model;
    private String vin;
    private int year;
    private double price;
}
